posthog/schema.py

```diff
@@ -2863,6 +2863,7 @@ class HogQLQueryModifiers(BaseModel):
     useMaterializedViews: Optional[bool] = None
     usePresortedEventsTable: Optional[bool] = None
     useWebAnalyticsPreAggregatedTables: Optional[bool] = None
+    convertToProjectTimezone: bool = True
```

Comment:
All modifiers should be `Optional` and the usage of them should have a fallback for when a modifier is not selected

Reply:
missed the default initialisation function, done it properly now 👍